java/app-encryption/src/main/java/com/godaddy/asherah/crypto/AeadCrypto.java

```diff
@@ -74,9 +74,9 @@ public CryptoKey generateKeyFromBytes(final byte[] sourceBytes) {
   /**
    * Generates a <code>CryptoKey</code> using the provided source bytes and created time. <b>NOTE</b>: you MUST wipe
    * out the source bytes after the completion of this call!
-   * @param sourceBytes
-   * @param created
-   * @return
+   * @param sourceBytes the source bytes to use.
+   * @param created the time to use for creation of <code>CryptoKey</code>.
+   * @return a generated CryptoKey.
```

Comment:
same as above. use the description for created from the generateRandomCryptoKey method